pkg/ddl/job_submitter.go

```diff
@@ -316,10 +316,9 @@ func (s *JobSubmitter) addBatchDDLJobs2Table(jobWs []*JobWrapper) error {
 	for _, jobW := range jobWs {
 		job := jobW.Job
 		if job.Version == 0 {
-			// if not set, fix it to version 1
-			// TODO replace this with assert after we add code v2 for all jobs.
-			job.Version = model.JobVersion1
+			return errors.Errorf("Job version should not be zero")
```

Comment:
replace it with assert is enough, here is to confirm that code in ddl/executor.go set Version explicitly